### ISSUE TITLE
重定向问题怎么解决呢？Http status error [302]

### ISSUE THREAD

**Issue by Mrlaugh** [flutterchina/dio#625]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by appdev** [comment]
这个问题仍然存在。dio还在继续维护吗？

**Comment by suink** [comment]
> 这个问题仍然存在。dio还在继续维护吗？

你是怎么解决的呀

**Comment by suink** [comment]
> 这个问题仍然存在。dio还在继续维护吗？

问题已解决，需要将dio.post(url, data, options)改为dio.post(url, data, options)
参看https://stackoverflow.com/questions/52537500/flutter-handle-status-code-302-in-post-request
